data/zachfrenesdev

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Zach Frenes",
+  "url": "https://github.com/zachfrenesdev",
+  "image": "",
```

Comment:
Kindly give the URL for your image.

Reply:
You can add an image "URL" by just pasting the link to your Github profile on a browser and adding .png at the end.
Example: https://github.com/cheloghm.png